R/mod_summary_table.R

```diff
@@ -0,0 +1,115 @@
+#' summary_table UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd
+#'
+#' @importFrom shiny NS tagList
+
+covid_data <- readxl::read_excel("inst/app/data/example_long_cvd.xlsx",
```

Comment:
In the long run we will factor out this data load into the rest of the package but this is fine for now